CMakeLists.txt

```diff
@@ -82,6 +82,7 @@ if(ENABLE_THEORA)
   add_definitions(-DTHEORA)
 endif()
 
+set(OpenGL_GL_PREFERENCE LEGACY)
```

Comment:
CMake policy CMP0072 should be set properly instead. See first lines of the file. And it's better to make it configurable instead.

Reply:
I'm newbie to cmake. How to make it configurable?

Reply:
```
cmake_minimum_required(VERSION 3.1.0)

set(LEGACY_OPENGL_LIBS OFF CACHE BOOL "Use legacy OpenGL libraries instead of glvnd library (Default: off)")

if (POLICY CMP0072)
  if (LEGACY_OPENGL_LIBS)
    cmake_policy(SET CMP0072 OLD)
  else()
    cmake_policy(SET CMP0072 NEW)
  endif()
endif()

...
```

Then you can run `cmake -DLEGACY_OPENGL_LIBS=ON ..`